upgrade-prechecks/preupgrade-healthcheck.sh

```diff
@@ -777,6 +777,30 @@ function verify_nodes_hw(){
   fi
 }
 
+function verify_nodes_fw(){
```

Comment:
@pruthvitd what is this check doing and useful as upgrade precheck ? Are we saying, if customer is not already on latest fw for installed level of Fusion, they should not start next version upgrade ?